solidity/contracts/vault/TBTCOptimisticMinting.sol

```diff
@@ -334,13 +334,12 @@ abstract contract TBTCOptimisticMinting is Ownable {
         uint256 optimisticMintFee = optimisticMintingFeeDivisor > 0
             ? amountToMint / optimisticMintingFeeDivisor
             : 0;
-        amountToMint -= optimisticMintFee;
```

Comment:
I think it's worth making it crystal-clear that the OM fee is actually optimistically minted as well so it must be added to the debt. This is not obvious when reading the code.

Reply:
Done in  653406ab32b95b8c0c520fb38dccb7d9e64a5e90. Please take a look and let me know if this explanation is enough.

Reply:
LGTM!